spec/models/result_spec.rb

```diff
@@ -115,6 +126,17 @@
                 it_returns 'empty'
               end
 
+              context 'when an invalid marking state is found' do
+                let!(:invalid_result) { create(:result, marking_state: Result::MARKING_STATES[:incomplete]) }
+                before do
+                  invalid_result.marking_state = 'wrong'
+                end
+
+                it 'catches an invalid result' do
```

Comment:
"catches" isn't a good verb here. Maybe "considers the result invalid". (Same below.)